OrganizationCompany.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I bet this is part of the org chart thing, so perhaps group these all in a folder and have instructions on how to use them together?